lib/pages/home_tab.dart

```diff
@@ -247,10 +247,21 @@ class HomeTab extends HookWidget {
                 iconTheme: IconThemeData(color: theme.colorScheme.onSurface),
                 titleTextStyle: theme.textTheme.titleLarge
                     ?.copyWith(fontSize: 20, fontWeight: FontWeight.w500),
-                leading: IconButton(
-                  icon: const Icon(Icons.notifications),
-                  onPressed: () => goTo(context, (_) => const InboxPage()),
-                ),
+                leading: accStore.notificationCount > 0
+                    ? Badge(
+                        offset: const Offset(-5, 5),
+                        label: Text(accStore.notificationCount.toString()),
```

Comment:
I forgot to mention this in my original review:
Shouldn't we have a limit on this? 
I.e. render `9+` instead potentially letting this get to triple digits and looking really ugly?